api/soap/mc_api.php

```diff
@@ -43,6 +43,11 @@
  */
 class SoapObjectsFactory {
 	/**
+	 * @var bool true: SOAP API, false: REST API
+	 */
+	static public $soap = true;
```

Comment:
I don't think this belong where. We should either unwrap the SoapVar in the REST API, as long as it's a transitional measure, or move this code someplace else.